File: PHP/8php 3/paperwork.php
<?php
/*
|--------------------------------------------------------------------------
| Beginner Series #1 School Paperwork
|--------------------------------------------------------------------------
|
*/

function paperwork(int $n, int $m)
{
   return $n < 0 || $m < 0 ? 0 : $n * $m;
}


function paperwork(int $n, int $m): int
{
  return max(0, $n) * max(0, $m);
}


function paperwork(int $n, int $m): int
{
   return $n > 0 ? ($m > 0 ? $n*$m : 0) : 0;
}


print_r(paperwork(5, 5). ' ' . 25 . '
');
print_r(paperwork(-5, 5). ' ' . 0 . '
');

File: Python/8py 3/paperwork.py
"""
|--------------------------------------------------------------------------
| Beginner Series #1 School Paperwork
|--------------------------------------------------------------------------
|
"""

def paperwork(n, m):
   return 0 if n < 0 or m < 0 else n * m


paperwork = lambda m,n: m*n if m>0 and n>0 else 0


def paperwork(n, m):
   if m<0 or n<0:
      return 0
   else:
      return n*m


def paperwork(n, m):
   return max(n, 0)*max(m, 0)


paperwork = lambda a,b: a*b if min(a,b)>0 else 0


def paperwork(n, m):
   return n * m if min((n, m)) > 0 else 0


print(paperwork(5,5), 25)
print(paperwork(5,-5), 0)